hs/app/reach/Main.hs

```diff
@@ -2517,12 +2516,12 @@ support = command "support" $ info (pure step1) d
       liftIO . T.putStrLn $ pack $ "Your user code is " ++ unpack userCode
       liftIO . T.putStrLn $ pack "Please enter it at https://github.com/login/device"
       liftIO . T.putStrLn $ pack ""
-      liftIO . T.putStrLn $ pack
-        "Type 'y' AFTER SUCCESSFUL AUTHORIZATION to upload index.mjs and index.rsh."
+      liftIO . T.putStrLn $ pack "Type 'y' AFTER SUCCESSFUL AUTHORIZATION"
```

Comment:
Why yelling?

Reply:
[a03d25e](https://github.com/reach-sh/reach-lang/pull/762/commits/a03d25ee2c3df9059ca8a81912ba7546c127c6c3) should address this.